test/integration/graphql_test.lua

```diff
@@ -1371,6 +1371,9 @@ function g.test_specifiedByUrl()
     }
 
     local data, errors = check_request(introspection.query, query_schema)
-    t.assert_equals(tostring(util.map_name(data.__schema.types, function(v) return v end)['CustomInt'].specifiedByUrl), 'http://localhost')
+    t.assert_equals(
+        tostring(util.map_name(data.__schema.types, function(v) return v end)['CustomInt'].specifiedByUrl),
+        'http://localhost'
+    )
```

Comment:
This commit should be squashed into previous one.

Reply:
?

Reply:
[This](https://github.com/tarantool/graphql/pull/21/commits/55e5d41ea7e160368388af5ce5f806c1c1c2ce4a) commit should be squashed into [previous one](https://github.com/tarantool/graphql/pull/21/commits/93a031ffdb05588c15f522c1d6eda31bc4fa48da).

Reply:
Why? It'a a test for SpecifiedByUrl field for custom Scalar. Please explain more detailed why this have to squashed into previous commit?

Reply:
I talk about commit "Fix lint warning".
All changes should be atomic. But here you introduces "Add Scalar specifiedByUrl field and specifiedByUrl directive" a code that breaks linter and fixes it the following commit.

"Add Scalar specifiedByUrl field and specifiedByUrl directive" should be self-sufficient and shouldn't request some follow-ups to fix it.

Reply:
Ok